playhand.js

```diff
@@ -419,17 +427,17 @@ function postflop(cardsString, boardCardsString, playersInHand, potSizeAtStartOf
     }
 
     // Open ended straight draw
-    if (hasStraightDraw(cards, boardCards) && !hasStraightDraw([], boardCards)) {
+    if (checkStraightOrDrawOfLength(cards.concat(boardCards), 4) && !checkStraightOrDrawOfLength(boadCards, 4)) {
```

Comment:
Not sure if you fixed this in a later PR... also not sure how vscode didn't catch this 🤷‍♀️ 
```suggestion
    if (checkStraightOrDrawOfLength(cards.concat(boardCards), 4) && !checkStraightOrDrawOfLength(boardCards, 4)) {
```